app/upgrades/v2/upgrades.go

```
@@ -0,0 +1,60 @@
+	errorsmod "cosmossdk.io/errors"
+	storetypes "cosmossdk.io/store/types"
+	upgradetypes "cosmossdk.io/x/upgrade/types"
+
+	"github.com/cosmos/cosmos-sdk/types/module"
+
+	wasmtypes "github.com/CosmWasm/wasmd/x/wasm/types"
+
+	"github.com/liftedinit/manifest-ledger/app/upgrades"
+)
+
+var (
```

golangci-lint (1.62.2)

File is not `gofumpt`-ed

(gofumpt)